how-to/access-isc-calendar.md

```diff
@@ -1,10 +1,39 @@
-The InnerSource Commons Foundation has a [Google calendar] that people can use to view and schedule events in the InnerSource Commons.
-If you want to use the calendar, then you can request access by posting a message in the [#google-admins] Slack channel.
+#Introduction to adding calendar events to our public calendar
+
+The InnerSource Commons Foundation has a public [Google calendar] that people can use to view and schedule events in the InnerSource Commons [website](https://innersourcecommons.org/calendar/).
+If you want to add an InnerSource event to our public calendar, you can request access by posting a message in the [#google-admins] Slack channel.
 The InnerSource Commons Public Calendar is for public events.
 The InnerSource Commons Private Calendar is for private events and should only be used by Foundation members.
 We expect that most events in the InnerSource Commons will be public.
 
 When creating events, be sure that your title is clear to anyone that reads it and doesn't contain some special jargon that is only understandable to people that are already in your group.
 
+Please follow this guidance when adding a new event to the calendar:
+
+1. Remove any abbreviations or jargon from the **title of your event** 
+2. Add date, time
+3. If you're having an online meeting, then please add the zoom link, eventbrite/registration link or how to access the meeting in the **where** section of the calendar notice
+4. The **meeting description** should be very clear:
+   - **Your first sentence should state whether the meeting/event is open to everyone or if it is open to people with an interest in a specific aspect of InnerSource.** 
+   - **Your next sentence should give a general description of the group.** 
+
+##Example 1 - Community Calls
+
+**Title**: Community Call: Why InnerSource works on Teams
+**When**: Tues, September 1, 5pm-6pm
+**Where**: See meeting description
+**Description**: 
+Our Community Calls are open to anyone interested in InnerSource. In our Community Calls, our experts speak on a topic and then answer questions from the community. In this Call Expert A will speak about their experience of managing successful InnerSource projects across teams in multinational companies and shares learning about what works.
+(Add link to zoom here)
+
+##Example 2 - Marketing Working Group
+
+**Title**: Marketing Working Group
+**When**: Tues, September 8, 4pm-5pm
+**Where**: (add jitsi link here)
+**Description**: 
+Our Marketing Working Groups are open to everyone in the InnerSource community. At our Marketing Working group, we discuss our activities promoting InnerSource to a wide audience. Our talks cover Community Calls, in-person Gatherings and official events. We're also currently discussing badging and recognition events. We welcome new ideas from everyone in the community.
+(Add link to meeting notes)
```

Comment:
```suggestion
## Example 1 - Community Calls

* **Title**: Community Call: Why InnerSource works on Teams
* **When**: Tues, September 1, 5pm-6pm
* **Where**: See meeting description
* **Description**: 
Our Community Calls are open to anyone interested in InnerSource. In our Community Calls, our experts speak on a topic and then answer questions from the community. In this Call Expert A will speak about their experience of managing successful InnerSource projects across teams in multinational companies and shares learning about what works.
(Add link to zoom here)

## Example 2 - Marketing Working Group

* **Title**: Marketing Working Group
* **When**: Tues, September 8, 4pm-5pm
* **Where**: (add jitsi link here)
* **Description**: 
Our Marketing Working Groups are open to everyone in the InnerSource community. At our Marketing Working group, we discuss our activities promoting InnerSource to a wide audience. Our talks cover Community Calls, in-person Gatherings and official events. We're also currently discussing badging and recognition events. We welcome new ideas from everyone in the community.
(Add link to meeting notes)
```
 If you look at the text in "view" mode it shows up all as one paragraph.  I think this will fix it (although I can't preview these changes, either).